docs/source/support/iroha.md

```diff
@@ -0,0 +1,17 @@
+Iroha Support
+-----------------
+
+```{note}
+The deployContract feature not yet implemented since Iroha lacks full smart contract support during the initial development stage of the Iroha connector plugin.
+```
+
+<details>
+  <summary>Hyperledger Cactus v0.8.0</summary>
```

Comment:
@hxlaf Please update to 0.9.0 (that's the earliest release that this will be able to be included in most likely)

Reply:
Fixed! Thank you!